source/common/event/dispatched_thread.cc

```diff
@@ -0,0 +1,50 @@
+#include "common/event/dispatched_thread.h"
+
+#include <chrono>
+
+#include "envoy/event/dispatcher.h"
+#include "envoy/event/timer.h"
+#include "envoy/server/configuration.h"
+
+#include "common/api/api_impl.h"
+#include "common/common/thread.h"
+
+#include "spdlog/spdlog.h"
+
+namespace Envoy {
+namespace Event {
+
+void DispatchedThreadImpl::start(Server::GuardDog& guard_dog) {
```

Comment:
I figured it out :) This is an excellent idea, I'll change out the no-exit-timers for a solution that uses EVLOOP_NO_EXIT_ON_EMPTY